tests/browsertype-connect.spec.ts

```diff
@@ -572,3 +573,57 @@ test('should fulfill with global fetch result', async ({ browserType, startRemot
   expect(response.status()).toBe(200);
   expect(await response.json()).toEqual({ 'foo': 'bar' });
 });
+
+test('should upload large file', async ({ browserType, startRemoteServer, server, browserName }, testInfo) => {
+  test.skip(browserName !== 'chromium');
+  test.slow();
+  const remoteServer = await startRemoteServer();
+  const browser = await browserType.connect(remoteServer.wsEndpoint());
+  const context = await browser.newContext();
+  const page = await context.newPage();
+
+  await page.goto(server.PREFIX + '/input/fileupload.html');
+  const uploadFile = testInfo.outputPath('200MB.zip');
+  const str = 'A'.repeat(4 * 1024);
+  const stream = fs.createWriteStream(uploadFile);
+  for (let i = 0; i < 50 * 1024; i++) {
```

Comment:
idea (might make not much sense): I personally would use a more complicated data payload, like counting numbers up and putting it in a string, by that we verify also the content in the data to spot off by one errors etc.

Reply:
I'd do that if I were to verify the content, so far it's just size comparison so should be good enough.